pkg/controller/v1alpha1/trainedmodel/controller.go

```diff
@@ -229,28 +230,47 @@ func (r *TrainedModelReconciler) updateConditions(req ctrl.Request, tm *v1alpha1
 		conditionErr = fmt.Errorf(InferenceServiceNotReady, isvc.Name, tm.Name)
 	}
 
-	// Update Framework Supported condition
 	isvcConfig, err := v1beta1api.NewInferenceServicesConfig(r.Client)
 	if err != nil {
 		return err
+	}
+
+	// Update Is MMS Predictor condition
+	implementations := isvc.Spec.Predictor.GetImplementations()
+	if len(implementations) > 0 && v1beta1utils.IsMMSPredictor(&isvc.Spec.Predictor, isvcConfig) {
+		log.Info("Predictor is multi-model serving", "TrainedModel", tm.Name, "InferenceService", isvc.Name)
```

Comment:
Is this logging needed? it can spew the log when you deploy large amount of trained models.

Reply:
I wanted to be consistent with the rest of the condition updates but I suppose on an Info level that would be excessive.